tests/conftest.py

```diff
@@ -1,32 +1,29 @@
+import logging
 import time
 from collections.abc import AsyncGenerator, Generator
 from textwrap import dedent
 from unittest.mock import AsyncMock, patch
 from uuid import uuid4
 
 import pytest
-import rich.console
 from opentelemetry.sdk.resources import Resource
 from opentelemetry.sdk.trace import ReadableSpan
 from opentelemetry.trace import SpanContext, SpanKind, TraceFlags, TraceState
 from opentelemetry.trace.status import Status, StatusCode
 
 from any_agent.config import AgentFramework
+from any_agent.logging import setup_logger
 
 
-@pytest.fixture(autouse=True)
```

Comment:
@njbrake this was not used for the logger but to avoid tracing output (the rich console usage) during test runs.

I don't think this change preserves the previously intended behavior

Reply:
So I did test it and with my changes it was correctly suppressing console output when `-s` was removed, and adding it in when `-s` was added. Was it supposed to do something else? (That's what I thought it was supposed to do)

Reply:
I don't understand what changed since I added the fixture but it seems that even if we remove your fixture the rich console gets correctly disabled. Maybe at some rich/pytest version this got fixed

Reply:
Noice. So I guess maybe we can remove the fixture completely. I'll test.